storefront/app/views/themes/default/spree/page_sections/_featured_taxon.html.erb

```diff
@@ -42,7 +42,7 @@
       </div>
     <% end %>
     <% if loaded && section.taxon.present? %>
-      <% if section.products(current_currency).any? %>
+      <% if taxon_products(current_currency, section).any? %>
```

Comment:
@brozek95 you're fetching this twice - on line 45 and 55, please store `taxon_products` in a variable